conv/addr.go

```diff
@@ -1,67 +0,0 @@
-package conv
```

Comment:
what happened to this? not needed?

Reply:
we do not need it since having specific param for bootstrap nodes, we can directly use peer addresses in multiaddress format, and no need to translate them from tendermint p2p format.

Reply:
ok great